app/models/DepartureStatus.scala

```diff
@@ -17,92 +17,92 @@
 package models
 
 sealed trait DepartureStatus {
-  def transition(messageReceived: MessageReceivedEvent): DepartureStatus
+  def transition(messageReceived: MessageReceivedEvent): Either[String, DepartureStatus]
```

Comment:
Can we specify an error type instead of String in the Either declaration?<br>Something like this:<br>`def transition(messageReceived: MessageReceivedEvent): Either[TransitionError, DepartureStatus]`<br><br>`sealed trait TransitionError`<br><br>`case class FailedToTransitionDeparture(reason: String) extends TransitionError`